app/src/pages/admin/allUsersPage/usersToolbar/usersToolbar.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+import { injectIntl, defineMessages, intlShape } from 'react-intl';
+import classNames from 'classnames/bind';
+import { GhostButton } from 'components/buttons/ghostButton';
+import ExportIcon from 'common/img/export-inline.svg';
+import InviteUserIcon from 'common/img/ic_invite-inline.svg';
+import AddUserIcon from 'common/img/ic-add-user-inline.svg';
+import { EXPORT, INVITE_USER, ADD_USER } from './constants';
+import styles from './usersToolbar.scss';
+
+const cx = classNames.bind(styles);
+
+const messages = defineMessages({
+  [EXPORT]: {
+    id: 'AllUsersPage.export',
+    defaultMessage: 'Export',
+  },
+  [INVITE_USER]: {
+    id: 'AllUsersPage.inviteUser',
+    defaultMessage: 'Invite user',
+  },
+  [ADD_USER]: {
+    id: 'AllUsersPage.AddUser',
+    defaultMessage: 'Add user',
+  },
+});
+
+const HEADER_BUTTONS = [
+  {
+    key: EXPORT,
+    icon: ExportIcon,
+    grayBorder: true,
+  },
+  {
+    key: INVITE_USER,
+    icon: InviteUserIcon,
+    grayBorder: false,
+  },
+  {
+    key: ADD_USER,
+    icon: AddUserIcon,
+    grayBorder: false,
+  },
+];
+
+@injectIntl
+export class UsersToolbar extends Component {
```

Comment:
Just a recommendation: this can be done using a functional component. Simply return the contents from the function if the component doesn't need any special methods or state.